### ISSUE TITLE
Flutter channel should sort channels by stability

### ISSUE THREAD

**Issue by Blaxou** [flutter/flutter#31014]
## Steps to Reproduce

1. Launch `flutter channel`
2. You get : 

```
Flutter channels:
  beta
  dev
  master
* stable
```
Where you should, logically get : 
```
Flutter channels:
  master
  dev
  beta
* stable
```
Making it clearer for everyone ;)

**Comment by mannprerak2** [comment]
Created a PR #48427 to resolve this

**Comment by mannprerak2** [comment]
This issue can be closed now... Fixed by #48427
